design/features/payment-links/version-1.md

```diff
@@ -0,0 +1,240 @@
+# Payment link v1
+
+## Status
+
+Date created: *2024-04-30*  
+
+Developed  
+
+___
+
+## Contents
+
+- [Status](#status)
+- [Contents](#contents)
+- [What](#what)
+- [Key decisions](#key-decisions)
+- [Designs](#designs)
+- [Preveiwing the form fillers journey](#preveiwing-the-form-fillers-journey)
+- [Research focus](#research-focus)
+
+___
+
+## What
+
+### As-is
+
```

Comment:
Do you think this needs to be added as the first point? Or do your updates to the points cover this now?

Reply:
Yeah sorry - the suggestion wasn't really as clear as I thought it would be! The suggestion as to add this as a first point as it's the main thing really? The other two points are sort of because of this. Does that make sense?

Reply:
Done